CMakeLists.txt

```diff
@@ -1307,10 +1307,10 @@ if(UNIX AND NOT APPLE)
       "${CMAKE_INSTALL_DATADIR}/pixmaps"
   )
 
-  # .appdata.xml file for KDE/GNOME AppStream initiative
+  # .metainfo.xml file for KDE/GNOME AppStream initiative
   install(
     FILES
-      "${CMAKE_CURRENT_SOURCE_DIR}/res/linux/mixxx.appdata.xml"
+      "${CMAKE_CURRENT_SOURCE_DIR}/res/linux/mixxx.metainfo.xml"
     DESTINATION
       "${CMAKE_INSTALL_DATAROOTDIR}/metainfo"
```

Comment:
This violates the rule for local installs not to change anything outside the install prefix. 
This also requires root, where it would be nice to install Mixxx at into the home directory without root access. 

Can we apply here the same hack we did for the udev rules?

Reply:
No, I am not planning to fix the CMake install issue.

I just stumbled over this legacy naming issue after the RPMFusion build broke due to the recent changes. This PR was only intended to improve consistency.